### PR TITLE
GitHub - part 5 (models and tutorials)

### PR DESCRIPTION
**Pull request by chapulina**
Update these 2 repos:

* https://github.com/osrf/gazebo_models
* https://github.com/osrf/gazebo_tutorials

### CODE REVIEW

File: color_model/tutorial_7.md [hunk 1]
```diff
@@ -221,14 +221,14 @@ Save this image as `head_texture.png`.
 
 [[file:files/head_texture.png|256]]
 
-[Download and save this Collada file as head.dae](http://bitbucket.org/osrf/gazebo_tutorials/raw/default/color_model/files/collada.dae).
+[Download and save this Collada file as head.dae](http://github.com/osrf/gazebo_tutorials/raw/master/color_model/files/collada.dae).
```
Comment (ahcorde) [inline color_model/tutorial_7.md:224]: This link doesn't exist

File: haptix_world_sim_api/tutorial.md [hunk 2]
```diff
@@ -58,15 +58,15 @@ Next, write a simple Octave/Matlab to sense contact state on the visual
...
 
 ## Get the code:
 
 Download tutorial files:
 
- - [custom_haptix.world](https://bitbucket.org/osrf/gazebo_tutorials/src/haptix_world_sim_api/haptix_world_sim_api/files/custom_haptix.world)
+ - [custom_haptix.world](https://github.com/osrf/gazebo_tutorials/blob/haptix_world_sim_api/haptix_world_sim_api/files/custom_haptix.world)
 
- - [custom_world.m](https://bitbucket.org/osrf/gazebo_tutorials/src/haptix_world_sim_api/haptix_world_sim_api/files/custom_world.m)
+ - [custom_world.m](https://github.com/osrf/gazebo_tutorials/blob/haptix_world_sim_api/haptix_world_sim_api/files/custom_world.m)
```
Comment (ahcorde) [inline haptix_world_sim_api/tutorial.md:69]: Link doesn't exist

File: haptix_world_sim_api/tutorial.md [hunk 3]
```diff
@@ -77,17 +77,17 @@ and write a simple C code to sense contact state on the visual
...
+<include lang='c' src='https://github.com/osrf/gazebo_tutorials/raw/master/haptix_world_sim_api/files/custom_world.c'/>
 
 Build `custom_world.c` as you would with any Haptix C API code as shown in [Compile section under the Haptix C API tutorial](http://gazebosim.org/tutorials?tut=haptix_comm&cat=haptix#Compileyourcontroller).
 
 ## Get the code:
 
 Download tutorial files:
 
- - [custom_haptix.world](https://bitbucket.org/osrf/gazebo_tutorials/src/haptix_world_sim_api/haptix_world_sim_api/files/custom_haptix.world)
+ - [custom_haptix.world](https://github.com/osrf/gazebo_tutorials/blob/haptix_world_sim_api/haptix_world_sim_api/files/custom_haptix.world)
```
Comment (ahcorde) [inline haptix_world_sim_api/tutorial.md:88]: link doesn't exist

File: haptix_world_sim_api/tutorial.md [hunk 4]
```diff
@@ -77,17 +77,17 @@ and write a simple C code to sense contact state on the visual
...
 
- - [custom_world.c](https://bitbucket.org/osrf/gazebo_tutorials/src/haptix_world_sim_api/haptix_world_sim_api/files/custom_world.c)
+ - [custom_world.c](https://github.com/osrf/gazebo_tutorials/blob/haptix_world_sim_api/haptix_world_sim_api/files/custom_world.c)
```
Comment (ahcorde) [inline haptix_world_sim_api/tutorial.md:90]: link doesn't exist

File: inertia/tutorial.md [hunk 5]
```diff
@@ -133,7 +133,7 @@ ranging from [1.604,1.618], which is close to the expected value.
 One thing to keep in mind is that duplicate faces within a mesh will affect
 the calculation of volume and moment of inertia.
 For example, consider another spherical mesh:
-[ball.dae](https://bitbucket.org/osrf/gazebo_models/raw/9f0a80a06cf3/robocup_3Dsim_ball/meshes/ball.dae).
+[ball.dae](https://github.com/osrf/gazebo_models/raw/9f0a80a06cf3/robocup_3Dsim_ball/meshes/ball.dae).
```
Comment (ahcorde) [inline inertia/tutorial.md:136]: Link doesn't exist

File: kinematic_loop/tutorial.md [hunk 6]
```diff
@@ -21,42 +21,42 @@ and 3 links named link\_AB, link\_BC, and link\_CD.
 
 ![screenshot of four\_bar_sdf model](https://bytebucket.org/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar.png)
 
-The model files are in the [four\_bar_sdf](https://bitbucket.org/osrf/gazebo_tutorials/src/kinematic_loop/kinematic_loop/four_bar_sdf)
+The model files are in the [four\_bar_sdf](https://github.com/osrf/gazebo_tutorials/blob/kinematic_loop/kinematic_loop/four_bar_sdf)
```
Comment (ahcorde) [inline kinematic_loop/tutorial.md:24]: Link doesn't exist

File: kinematic_loop/tutorial.md [hunk 7]
```diff
@@ -21,42 +21,42 @@ and 3 links named link\_AB, link\_BC, and link\_CD.
...
 folder.
 To use this model, create a folder `~/.gazebo/models/four_bar_sdf` and copy the
-[model.config](https://bitbucket.org/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_sdf/model.config)
+[model.config](https://github.com/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_sdf/model.config)
```
Comment (ahcorde) [inline kinematic_loop/tutorial.md:27]: Link doesn't exist

File: kinematic_loop/tutorial.md [hunk 8]
```diff
@@ -21,42 +21,42 @@ and 3 links named link\_AB, link\_BC, and link\_CD.
...
 and
-[model.sdf](https://bitbucket.org/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_sdf/model.sdf)
+[model.sdf](https://github.com/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_sdf/model.sdf)
```
Comment (ahcorde) [inline kinematic_loop/tutorial.md:29]: Link doesn't exist

File: kinematic_loop/tutorial.md [hunk 9]
```diff
@@ -21,42 +21,42 @@ and 3 links named link\_AB, link\_BC, and link\_CD.
...
 to that folder.
 You can then insert the model into a simulation using the `Insert` panel
 in the [left side of the gazebo client](http://gazebosim.org/tutorials?cat=guided_b&tut=guided_b2).
 
 For brevity, the model parameters are encoded in an embedded ruby template file named
-[model.sdf.erb](https://bitbucket.org/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_sdf/model.sdf.erb).
+[model.sdf.erb](https://github.com/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_sdf/model.sdf.erb).
```
Comment (ahcorde) [inline kinematic_loop/tutorial.md:35]: Link doesn't exist

File: kinematic_loop/tutorial.md [hunk 10]
```diff
@@ -21,42 +21,42 @@ and 3 links named link\_AB, link\_BC, and link\_CD.
...
 The model.sdf can be generated from the template using the `erb` command: `erb -T 1 model.sdf.erb > model.sdf`.
 
 This allows geometric parameters to be defined in one place
 along with a helper function for computing the moment of inertia of a uniform box.
 
-<include lang='ruby' from='/  # Box dimensions/' to='/  # Points/' src='https://bitbucket.org/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_sdf/model.sdf.erb' />
+<include lang='ruby' from='/  # Box dimensions/' to='/  # Points/' src='https://github.com/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_sdf/model.sdf.erb' />
```
Comment (ahcorde) [inline kinematic_loop/tutorial.md:41]: Link doesn't exist

File: kinematic_loop/tutorial.md [hunk 11]
```diff
@@ -21,42 +21,42 @@ and 3 links named link\_AB, link\_BC, and link\_CD.
...
 
 The link parameters are stored in a dictionary named `boxes`:
 
-<include lang='ruby' from='/  # Points/' to='/  # Revolute/' src='https://bitbucket.org/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_sdf/model.sdf.erb' />
+<include lang='ruby' from='/  # Points/' to='/  # Revolute/' src='https://github.com/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_sdf/model.sdf.erb' />
```
Comment (ahcorde) [inline kinematic_loop/tutorial.md:45]: Link doesn't exist

File: kinematic_loop/tutorial.md [hunk 12]
```diff
@@ -21,42 +21,42 @@ and 3 links named link\_AB, link\_BC, and link\_CD.
...
 
 and the joint parameters are stored in a dictionary named `joints`:
 
-<include lang='ruby' from='/  # Revolute/' to='/  # end first ruby code block/' src='https://bitbucket.org/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_sdf/model.sdf.erb' />
+<include lang='ruby' from='/  # Revolute/' to='/  # end first ruby code block/' src='https://github.com/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_sdf/model.sdf.erb' />
```
Comment (ahcorde) [inline kinematic_loop/tutorial.md:49]: Link doesn't exist

File: kinematic_loop/tutorial.md [hunk 13]
```diff
@@ -21,42 +21,42 @@ and 3 links named link\_AB, link\_BC, and link\_CD.
...
 
 A model template is then included that references the computed parameters
 for each link and joint:
 
-<include lang='xml' from='/.sdf version/' to='@/sdf.@' src='https://bitbucket.org/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_sdf/model.sdf.erb' />
+<include lang='xml' from='/.sdf version/' to='@/sdf.@' src='https://github.com/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_sdf/model.sdf.erb' />
```
Comment (ahcorde) [inline kinematic_loop/tutorial.md:54]: Link doesn't exist

File: kinematic_loop/tutorial.md [hunk 14]
```diff
@@ -21,42 +21,42 @@ and 3 links named link\_AB, link\_BC, and link\_CD.
...
 
-The full [model.sdf](https://bitbucket.org/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_sdf/model.sdf)
+The full [model.sdf](https://github.com/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_sdf/model.sdf)
```
Comment (ahcorde) [inline kinematic_loop/tutorial.md:56]: Link doesn't exist

File: kinematic_loop/tutorial.md [hunk 15]
```diff
@@ -21,42 +21,42 @@ and 3 links named link\_AB, link\_BC, and link\_CD.
...
 is instantiated from this template, as you can see in the snippet below:
 
-<include lang='xml' from='/    .link name="link_CD"./' to='/joint_B. type=.revolute../' src='https://bitbucket.org/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_sdf/model.sdf' />
+<include lang='xml' from='/    .link name="link_CD"./' to='/joint_B. type=.revolute../' src='https://github.com/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_sdf/model.sdf' />
```
Comment (ahcorde) [inline kinematic_loop/tutorial.md:59]: Link doesn't exist

File: kinematic_loop/tutorial.md [hunk 16]
```diff
@@ -74,15 +74,15 @@ and a fixed joint joint\_E.
 
 ![screenshot of four\_bar\_split\_fixed_sdf model](https://bytebucket.org/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_split.png)
 
-The SDFormat model files are in the [four\_bar\_split\_fixed_sdf](https://bitbucket.org/osrf/gazebo_tutorials/src/kinematic_loop/kinematic_loop/four_bar_split_fixed_sdf)
+The SDFormat model files are in the [four\_bar\_split\_fixed_sdf](https://github.com/osrf/gazebo_tutorials/blob/kinematic_loop/kinematic_loop/four_bar_split_fixed_sdf)
```
Comment (ahcorde) [inline kinematic_loop/tutorial.md:77]: Link doesn't exist

File: kinematic_loop/tutorial.md [hunk 17]
```diff
@@ -74,15 +74,15 @@ and a fixed joint joint\_E.
...
 folder.
 To use this model, create a folder `~/.gazebo/models/four_bar_split_fixed_sdf` and copy the
-[model.config](https://bitbucket.org/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_split_fixed_sdf/model.config)
+[model.config](https://github.com/osrf/gazebo_tutorials/raw/kinematic_loop/kinematic_loop/four_bar_split_fixed_sdf/model.config)
```
Comment (ahcorde) [inline kinematic_loop/tutorial.md:80]: review links of this tutorial

**Comment by chapulina** [comment]
Thanks, @ahcorde , I believe I've fixed all the broken links you found on 89b6c3e00b409ff3610302fabfc36572b382da64

**Pull request merged**